src/app.js

```diff
@@ -23,7 +24,28 @@ app.use(bodyParser.json())
 const PORT = process.env.PORT || 3000
 const HOST = process.env.HOST || 'localhost'
 
+// run cache
+let INITIALIZED
+
+function checkIfInitialized(res) {
+  if (!INITIALIZED) return res.status(401).send('Error: cannot execute without initialization.')
```

Comment:
this function is also called on `cleanup`. so change the error message to something different.